src/please.go

```diff
@@ -476,7 +480,24 @@ var buildFunctions = map[string]func() int{
 		}
 
 		shouldSandbox := state.Graph.TargetOrDie(opts.Exec.Args.Target).Sandbox
-		return exec.Exec(state, opts.Exec.Args.Target, opts.Exec.Args.OverrideCommandArgs, process.NewSandboxConfig(shouldSandbox && !opts.Exec.Share.Network, shouldSandbox && !opts.Exec.Share.Mount))
+		dir := filepath.Join(core.OutDir, "exec", opts.Exec.Args.Target.Subrepo, opts.Exec.Args.Target.PackageName)
+		if code := exec.Exec(state, opts.Exec.Args.Target, dir, opts.Exec.Args.OverrideCommandArgs, process.NewSandboxConfig(shouldSandbox && !opts.Exec.Share.Network, shouldSandbox && !opts.Exec.Share.Mount)); code != 0 {
+			return code
+		}
+
+		for _, out := range opts.Exec.Output.Output {
+			from := filepath.Join(dir, out)
+			to := filepath.Join(opts.Exec.Output.OutputPath, out)
+
+			if err := fs.EnsureDir(to); err != nil {
+				log.Fatalf("%v", err)
+			}
+
+			if err := fs.RecursiveLink(from, to, 0644); err != nil {
+				log.Fatalf("failed to move output: %v", err)
+			}
+		}
```

Comment:
Can't decide whether it makes sense for this to be here or inside `exec.Exec`. Perhaps here is the right choice.

Reply:
Yeah I know what you mean. It feels like it could be merged with the logic from `plz test` but it's hard to tell until we actually get `plz test` to use this.